pkg/common/adaptor/response.go

```diff
@@ -54,7 +54,7 @@ func (c *compatResponse) WriteHeader(statusCode int) {
 				}
 				if k == consts.HeaderSetCookie {
 					cookie := protocol.AcquireCookie()
-					cookie.Parse(vv)
+					_ = cookie.Parse(vv)
```

Comment:
Error returned is ignored. This is likely acceptable because, even when an error is returned, all fields in the Cookie have already been appropriately set.